src/NServiceBus.Persistence.AzureStorage.AcceptanceTests/Sagas/When_doing_request_response_between_sagas.cs

```diff
@@ -54,7 +54,7 @@ protected override void ConfigureHowToFindSaga(SagaPropertyMapper<RequestRespons
 
                 public class RequestResponseRequestingSagaData : ContainSagaData
                 {
-                    public virtual Guid CorrIdForResponse { get; set; } //wont be needed in the future
+                    public virtual Guid CorrIdForResponse { get; set; }
```

Comment:
I digged through the history of this tests, including the rename after changing folder structure. It looks like the intial commit of this is here: https://github.com/Particular/NServiceBus.Persistence.AzureStorage/commit/d3342af56b720aa69e9c60bffeb06901a09f0eaf . This is dated Dec 2015. I'd day that we should drop the comment according to our current understanding and leave this property as an existing but no longer assigned correlation id.
What's your opinion?